services/web/server/src/simcore_service_webserver/login/_constants.py

```diff
@@ -1,6 +1,9 @@
 from typing import Final
 
 MSG_2FA_CODE_SENT: Final[str] = "Code sent by SMS to {phone_number}"
+MSG_2FA_UNAVAILABLE_OEC: Final[
+    str
+] = "Currently we cannot use 2FA, please try again later ({error_code})"
```

Comment:
make it impersonal
```suggestion
] = "2FA cannot be currently used, please try again later ({error_code})"
```

Reply:
app-teams' message ... delicate to change ;-)